Omise.Examples/Examples/Disputes.cs

```diff
@@ -45,17 +45,21 @@ public class Disputes : Example
         public async Task Update__Update()
         {
             var dispute = RetrieveOpenDispute();
-            dispute = await Client.Disputes.Update(dispute.Id, new UpdateDisputeRequest
-            {
-                Message = "Hello, World!"
-            });
 
-            Console.WriteLine($"updated dispute: {dispute.Id}");
+            if (dispute != null)

```

Comment:
Personally do not like this style of code. This only gives us the peace of mind but not the tool to fix it or the nag to tell us there is a problem. This works like an empty `catch() { }` clause.

If we expect the `dispute` to be there, then we should expect exceptions to be raised if it wasn't krub. I'd say let's not do this or suggest our merchant to do this. I think there is a better way.

---

On a side note, if you want to test disputes, just pop onto omise dashboard and open some manually. You can do that in test mode. This test was written with the that assumption in mind krub.

Reply:
Ah yes, this was just a hack to get the Example to complete 😅 - I'll add a catch into the runMethod to handle the missing data, and remove this hack